packages/catalog-process/src/services/catalogService.ts

```diff
@@ -2184,7 +2184,9 @@ export function catalogServiceBuilder(
 
       const validatedRiskAnalysisForm = validateRiskAnalysisSchemaOrThrow(
         eserviceRiskAnalysisSeed.riskAnalysisForm,
-        tenant.kind
+        tenant.kind,
+        new Date() // [todo remove comment] risk analysis creation
```

Comment:
I think that if we default to `new Date` or just use `new Date` always in the check (see comment below), we solve these drawbacks since we don't have to pass these params here